docs/modules/ROOT/pages/documentation/payara-server/server-configuration/http/protocols.adoc

```diff
@@ -13,3 +13,604 @@ commands. The protocol configurations can be found in the admin console under
+=== DNS Lookup
+
+Enables Domain Name System lookups.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.dns-lookup-enabled=[true/false]`
+
+[[configuration-http-trace-operation]]
+=== Trace Enabled
+
+Enables the TRACE operation. Setting this to false helps prevent cross-site scripting attacks.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.trace-enabled=[true/false]`
+
+[[configuration-http-auth-pass-through]]
+=== Auth Pass Through
+
+Enable this if the network listener receives traffic from an SSL terminating proxy server.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.auth-pass-through-enabled=[true/false]`
+
+[[configuration-http-chunking-enabled]]
+=== Chunking Enabled
+
+Enables HTTP response chunking.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.chunking-enabled=[true/false]`
+
+[[configuration-http-xpowered-by]]
+=== XPoweredBy
+
+Whether to include the X-Powered-By header in responses.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.xpowered-by=value`
+
+[[configuration-http-xframe-options]]
+=== XFrame Options
+
+When enabled, the X-Frame-Options header will be set to SAMEORIGIN. This can still be overrided by
+a deployed application.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.xframe-options=value`
+
+[[configuration-http-encoded-slash]]
+=== Encoded Slash
+
+Allow encoded slashes (e.g. `%2F`) in URIs.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.encoded-slash-enabled=[true/false]`
+
+[[configuration-http-websockets-support]]
+=== Websockets Support Enabled
+
+Allows websockets to be used.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.websockets-support-enabled=[true/false]`
+
+[[configuration-http-scheme-mapping]]
+=== Scheme Mapping
+
+HTTP header name used for identifying the originating protocol of a HTTP request.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.scheme-mapping=value`
+
+[[configuration-http-remote-user-mapping]]
+=== Remote User Mapping
+
+HTTP header name used for identifying the originating user of a HTTP request.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.remote-user-mapping=value`
+
+== HTTP/2 Options
+
+The following HTTP/2 fields can be configured for protocols:
+
+[[configuration-http-http2]]
+=== HTTP/2 Enabled
+
+Whether to enable HTTP/2 support for the protocol. Note that HTTP/2 can only be used on a secure listener.
```

Comment:
The second sentence should be in an admonition:

IMPORTANT: HTTP/2 support can only be enabled on a secure network listener